src/Tools/dotnet-monitor/Egress/EgressProviderTypes.cs

```diff
@@ -8,6 +8,8 @@ internal static class EgressProviderTypes
     {
         public const string AzureBlobStorage = nameof(AzureBlobStorage);
 
+        public const string External = nameof(External);
```

Comment:
This shouldn't be statically known since extensions will register their custom types via existence in the file system.